struct_guild.go

```diff
@@ -14,8 +14,21 @@ import (
 // consts inspired by: https://github.com/bwmarrin/discordgo/blob/master/structs.go
 
 // Constants for the different bit offsets of text channel permissions
+//go:generate stringer -type Permission
+type Permission uint
+
 const (
-	PermissionReadMessages = 1 << (iota + 10)
+	PermissionCreateInstantInvite Permission = 1 << iota
+	PermissionKickMembers
+	PermissionBanMembers
+	PermissionAdministrator
+	PermissionManageChannels
+	PermissionManageGuild
+	PermissionAddReactions
+	PermissionViewAuditLog
+	PermissionPrioritySpeaker
+	_
```

Comment:
reduce cognitive load by dropping these skippers and creating a new const block.